pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -417,35 +425,66 @@ func getInstallerPodName(revision int32, nodeName string) string {
 	return fmt.Sprintf("installer-%d-%s", revision, nodeName)
 }
 
-// ensureInstallerPod creates the installer pod with the secrets required to if it does not exist already
-func (c *InstallerController) ensureInstallerPod(nodeName string, operatorSpec *operatorv1.OperatorSpec, revision int32) error {
-	required := resourceread.ReadPodV1OrDie([]byte(installerPod))
-	required.Name = getInstallerPodName(revision, nodeName)
-	required.Namespace = c.targetNamespace
-	required.Spec.NodeName = nodeName
-	required.Spec.Containers[0].Image = c.installerPodImageFn()
-	required.Spec.Containers[0].Command = c.command
-	required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args,
-		fmt.Sprintf("-v=%d", 4),
+// toJSONArray converts a string array into JSON format that can be used in array field in YAML (args, command, etc.)
+// Example: []string{"1", "2"} will convert to: '["1","2"]'
+func toJSONArray(arr []string) string {
```

Comment:
This would still be nice to have as a helper somewhere if we want to switch away from the inline pod definitions

Reply:
agree, probably something in assets package or new "string" package.